cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/instantsearch-config.js

```diff
@@ -232,13 +237,19 @@ function enableInstantSearch(config) {
                     showMoreText: algoliaData.strings.moreResults,
                     empty: '',
                     item(hit, { html, components }) {
+                        const displayCalloutMsg = !hit.calloutMsg && 'd-none';
```

Comment:
Naming: this class will be used for *hidding* the callout message, not displaying it.

(Sorry to be picky but those details really help to read and understand the code)